io_scene_godot/converters/armature.py

```diff
@@ -123,13 +123,20 @@ def export_armature_node(escn_file, export_settings, node, parent_gd_node):
     skeleton_node = SkeletonNode(node.name, parent_gd_node)
     skeleton_node['transform'] = node.matrix_local
 
-    bone_list = list()
+    # according to configures, generate a set of blender bones
+    # need to be exported
+    bl_bones_to_export = list()
```

Comment:
here the `list()` can be `set()`, maybe it will improve some performance, but only affect when there are a large amount of bones.     I didn't use `set()` because in that case the order of bone would change and we need to recreate our test cases